### ISSUE TITLE
Custom Query Filter Not Working in Posts Widget

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#15074]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

I followed https://developers.elementor.com/custom-query-filter/ to create a custom query that I confirmed was added to my active theme's functions.php file.

```
// Showing multiple post types in Posts Widget
add_action( 'elementor/query/all_posts_no_faqs', function( $query ) {
	// Here we set the query to fetch posts with
	// post type of 'custom-post-type1' and 'custom-post-type2'
	$query->set( 'post_type', [ 'guides', 'newsletters', 'videos' ] );
} );
```

In the posts widget's current query query ID, I entered `all_posts_no_faqs`. No posts display...

No idea what to try.

**Comment by besweeet** [comment]
Yes. I confirmed they are correct by basically pulling up the wp-admin page that lists the "posts" and I note the slug at the end of the URL. For example: `/wp-admin/edit.php?post_type=guides`

**Comment by vgstef** [comment]
I've try for hours and it just seems that "Current query" doesn't take into consideration the custom query.

Anyone succeeded to add a custom query ID to a posts widget with current query?

**Comment by sanasahban** [comment]
I am facing the same issue. After trying for hours, I realized that the elementor post widget is not taking my Custom Query into account. 

I need to filter the posts being loaded into the posts widget, embedded in the content area of my custom post type. Looking forward to a workable solution without installing any new plugin.

**Comment by lucasvlc** [comment]
Having the same problem here. I'm using the following, where xxx is my query id:
```
add_action( 'elementor/query/xxx', function ( $query ) {
    $query->set( 'post_type', array( 'proyectos', 'publicacion' ) );
} );
```
Am I missing something?
